keyboards/ydkb/mountain/info.json

```diff
@@ -0,0 +1,170 @@
+{
+    "manufacturer": "YDKB",
+    "keyboard_name": "Mountain Keyboard",
+    "maintainer": "Wraul",
+    "bootloader": "lufa-ms",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "D5"
+    },
+    "matrix_size": {
+        "cols": 16,
+        "rows": 5
+    },
```

Comment:
```suggestion
```

Reply:
I don't understand. If I remove the "matrix_size" it won't compile?
Am I missing something?

```
make ydkb/mountain:default
QMK Firmware 0.23.9
☒ 'matrix_size'
Traceback (most recent call last):
  File "/nix/store/9h4bn3gg1qn5byjvbyffmv8q1p8289bq-python3-3.10.11-env/lib/python3.10/site-packages/milc/milc.py", line 528, in __call__
    return self.__call__()
  File "/nix/store/9h4bn3gg1qn5byjvbyffmv8q1p8289bq-python3-3.10.11-env/lib/python3.10/site-packages/milc/milc.py", line 533, in __call__
    return self._subcommand(self)
  File "/home/mathias/Developer/qmk_firmware/lib/python/qmk/decorators.py", line 27, in wrapper
    return func(*args, **kwargs)
  File "/home/mathias/Developer/qmk_firmware/lib/python/qmk/cli/list/layouts.py", line 21, in list_layouts
    info_data = info_json(cli.config.list_layouts.keyboard)
  File "/home/mathias/Developer/qmk_firmware/lib/python/qmk/info.py", line 216, in info_json
    info_data = _extract_led_config(info_data, str(keyboard))
  File "/home/mathias/Developer/qmk_firmware/lib/python/qmk/info.py", line 706, in _extract_led_config
    cols = info_data['matrix_size']['cols']
KeyError: 'matrix_size'
Making ydkb/mountain with keymap default

☒ 'matrix_size'
Traceback (most recent call last):
  File "/nix/store/9h4bn3gg1qn5byjvbyffmv8q1p8289bq-python3-3.10.11-env/lib/python3.10/site-packages/milc/milc.py", line 528, in __call__
    return self.__call__()
  File "/nix/store/9h4bn3gg1qn5byjvbyffmv8q1p8289bq-python3-3.10.11-env/lib/python3.10/site-packages/milc/milc.py", line 533, in __call__
    return self._subcommand(self)
  File "/home/mathias/Developer/qmk_firmware/lib/python/qmk/cli/generate/rules_mk.py", line 65, in generate_rules_mk
    kb_info_json = dotty(info_json(cli.args.keyboard))
  File "/home/mathias/Developer/qmk_firmware/lib/python/qmk/info.py", line 216, in info_json
    info_data = _extract_led_config(info_data, str(keyboard))
  File "/home/mathias/Developer/qmk_firmware/lib/python/qmk/info.py", line 706, in _extract_led_config
    cols = info_data['matrix_size']['cols']
KeyError: 'matrix_size'

 *                                                                                                     [ERRORS]
```

Reply:
`matrix_size` is not part of the `info.json` schema, though it is used internally. For custom matrix, the matrix dimensions need to be specified in `config.h`.

Reply:
I see. Thank you for the explanation.

This should hopefully be fixed now.